<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Blending</class>
 <widget class="QDialog" name="Blending">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1014</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Image 1 :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="ImageOneLineEdit"/>
     </item>
     <item>
      <widget class="QSlider" name="alphaSlider">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Image 2 : </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="ImageTwoLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QGraphicsView" name="pixelDataGraphics"/>
     <widget class="QListWidget" name="imageListWidget"/>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
